File: app/src/main/java/com/example/jacob/myapplication/DeepAnalysis/SentimentData.java
package com.example.jacob.myapplication.DeepAnalysis;

/**
 * Created by jacob on 25/11/2015.
 */
public abstract class SentimentData {

    float aggresiveness;
    float love;
    float happiness;
    float fear;
    float sadness;

    public SentimentData(float aggresiveness, float fear, float happiness, float love, float sadness) {
        this.aggresiveness = aggresiveness;
        this.fear = fear;
        this.happiness = happiness;
        this.love = love;
        this.sadness = sadness;
    }

    @Override
    public String toString() {
        return "SentimentData{" +
                "aggresiveness=" + aggresiveness +
                ", love=" + love +
                ", happiness=" + happiness +
                ", fear=" + fear +
                ", sadness=" + sadness +
                '}';
    }

    public float getAggresiveness() {
        return aggresiveness;
    }

    public float getFear() {
        return fear;
    }

    public float getHappiness() {
        return happiness;
    }

    public float getLove() {
        return love;
    }

    public float getSadness() {
        return sadness;
    }
}

File: app/src/main/java/com/example/jacob/myapplication/DeepAnalysis/SentimentData.kt
package com.example.jacob.myapplication.DeepAnalysis

/**
 * Created by jacob on 25/11/2015.
 */
abstract class SentimentData(aggressiveness: Float, fear: Float, happiness: Float, love: Float, sadness: Float) {

    var aggressiveness: Float = 0.toFloat()
        internal set
    var love: Float = 0.toFloat()
        internal set
    var happiness: Float = 0.toFloat()
        internal set
    var fear: Float = 0.toFloat()
        internal set
    var sadness: Float = 0.toFloat()
        internal set

    init {
        this.aggressiveness = aggressiveness
        this.fear = fear
        this.happiness = happiness
        this.love = love
        this.sadness = sadness
    }
}
